### PR TITLE
Add Readme support to playground pages

### PR DESCRIPTION
**Pull request by grapegravity**
Each component can now display its own Readme alongside its Example. This is formatted with Markdown and should eventually have most of the same functionality as GitHub's Markdown support.

screen shot 2015-02-18 at 2 56 48 pm

This has been tweaked to be a bit more readable, and a few more READMEs have been added.

### CODE REVIEW

**Comment by nelix** [comment]
:+1:

**Comment by grapegravity** [comment]
Is this sufficient to merge this, or should I work more on the documentation?

File: packages/avatar-with-pie/example.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ module.exports = React.createClass({
         <AvatarWithPie complete="3" total="6" email="james.coleman@bugherd.com" firstName="James" lastName="Coleman" size="m" />
         <AvatarWithPie complete="5" total="6" email="matt@bugherd.com" firstName="Matt" lastName="Milosavljevic" size="s" circle={true} />
 
-        <div style={{backgroundColor: '#21323a', color: '#fff', marginLeft: '-1em', marginRight: '-1em', paddingLeft: '1em', paddingRight: '1em'}}>
+        <div style={{backgroundColor: '#21323a', color: '#fff', marginLeft: '-30px', marginRight: '-30px', paddingLeft: '30px', paddingRight: '30px'}}>
```
Comment (nelix) [inline packages/avatar-with-pie/example.jsx]: px is implicit in react style objects

File: packages/icon/README.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+/>
+```
+
+## Properties
+
+### `type`
+
+The type of icon. Available types are;
+
+* `action-redo`
```
Comment (nelix) [inline packages/icon/README.md]: its a shame we can't use magic enumeration inside our docs :(

File: packages/avatar-with-pie/example.jsx (hunk 1 above)
Comment (grapegravity) [inline packages/avatar-with-pie/example.jsx]: I don’t really like that; explicit over implicit, but okay.

**Comment by nelix** [comment]
This is quite good, I'd like to merge this.
Adding any more docs to this PR just increases the chances that the person reviewing the PR won't read them.

File: packages/icon/README.md (hunk 2 above)
Comment (grapegravity) [inline packages/icon/README.md]: How do you mean?

Comment (nelix) [inline packages/icon/README.md]: I mean its a shame to have to maintain two lists of icon types

Comment (grapegravity) [inline packages/icon/README.md]: Ah, yes, well, I’d rather it were plaintext so it shows up on GitHub. :wink:

Comment (nelix) [inline packages/icon/README.md]: both those concerns are not mutually exclusive, but I think its not worth the effort until we have a more interesting release cycle and gh-pages (and the icon update freq becomes so intense we simply cannot keep up any more!)

**Pull request merged**